package.json

```diff
@@ -0,0 +1,67 @@
+{
```

Comment:
- [ ] [Redux Logger](https://www.npmjs.com/package/redux-logger) is missing from your package. Kindly note that this activity requires that you make use `redux-logger` package. Kindly ensure to add/use the same.